src/libraries/Microsoft.PowerFx.Core/Functions/IAsyncTexlFunction5.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Functions
+{    
+    internal interface IAsyncTexlFunction5
```

Comment:
>IAsyncTexlFunction5 [](http://example.com/codeflow?start=23&length=19)

We really, **really** need to stop creating more of those IAsyncTexlFunctionX...

Reply:
I know... already discussed w/ Mike
This one is probably going to be the last one